R/Connect.R

```diff
@@ -82,6 +82,57 @@ createConnectionDetails <- function(dbms,
 }
 
 jdbcDrivers <- new.env()
+jdbcDrivers$defaultPath <- Sys.getenv("DATABASECONNECTOR_JAR_FOLDER", 
+                                      unset = path.expand("~/DatabaseConnectorDrivers"))
+
+#' Download DatabaseConnector JDBC Jar files
+#' 
+#' Download the DatabaseConnector JDBC drivers from https://github.com/OHDSI 
+#'
+#' @param destination The full path to the folder where the Jar files should be downloaded to.
+#'        If NULL (default) then the value of the environment variable "DATABASECONNECTOR_JAR_FOLDER" is used.
+#'        If DATABASECONNECTOR_JAR_FOLDER is not set then "~/DatabaseConnectorDrivers" is used.
+#' @param method The method used for downloading files. See \code{?download.file} for details and options.
+#' @param ... Further arguments passed on to \code{download.file} 
+#' 
+#' @return Invisibly returns the destination if the download was successful.
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' downloadJdbcDrivers()
+#' }
+downloadJdbcDrivers <- function(destination = NULL, method = "auto", ...){
+  if (is.null(destination) || missing(destination)) {
```

Comment:
Should we allow the user to specify which JDBC driver to download? Likely they'll only need one.

Reply:
Yes. Agreed.